doc/source/whatsnew/v3.0.0.rst

```diff
@@ -102,8 +102,10 @@ Deprecations
 Removal of prior version deprecations/changes
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 - Removed :meth:`DataFrameGroupby.fillna` and :meth:`SeriesGroupBy.fillna` (:issue:`55719`)
+- Removed ``axis`` argument from :meth:`DataFrame.groupby` and :meth:`Series.groupby` (:issue:`51203`)
```

Comment:
Could you mention rolling and resample too?

Reply:
Thanks, done.